patientIdentifierType.md

```diff
@@ -0,0 +1,44 @@
+# PatientIdentifierType
+
+## Overview
+Administrators define what types of identifiers they will collect. These range from National ID numbers, to driver's license numbers, to per-hospital medical record numbers.
+
+### Properties on PatientIdentifierType:
+
+* format: a regular expression defining what the identifier text should contain.
+
+* formatDescription: the text an admin can enter describing the regex format they just added
+
+* required: a true/false whether every patient MUST have this type
+
+* scheckDigit: a true/false whether this identifier has a checkdigit at the end.
+
+* validator: full class name of the IdentifierValidators
```

Comment:
The full class name of an`org.openmrs.patient.IdentifierValidator` responsible for validating this type of identifiers.